src/spf-explainer/templates/app.vue

```diff
@@ -140,7 +145,7 @@ limitations under the License.
                 }
 
                 this.$data.records = records
-                window.history.pushState({}, "", `?domain=${domain}`)
+                window.history.replaceState({}, "", `?domain=${domain}`)
```

Comment:
This no longer inserts a new history item for each search, which defeats the entire point of adding support for the back button?

Reply:
This is still an issue, works fine on DNS, not here, due to this.